postgresql-plugin/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -55,3 +55,56 @@ datatypesSchema=[{"key":"id","value":"string"},{"key":"col1","value":"string"},{
   {"key":"col52","value":"string"},{"key":"col53","value":"string"},{"key":"col54","value":"string"},\
   {"key":"col55","value":"string"},{"key":"col56","value":"string"},{"key":"col57","value":"string"},\
   {"key":"col58","value":"int"},{"key":"col59","value":"int"},{"key":"col60","value":"long"}]
+
+#POSTGRESQL Invalid Properties
+invalidRef=invalidRef&^*&&*
+invalidDatabaseName=invalidDB
+invalidImportQuery=select
+invalidTableNameImportQuery=select * from abc;
+invalidDriverName=abcdriver
+invalidHost=testhost
+invalidPort=testPort
+invalidUserName=testUser
+invalidPassword=testPassword
+invalidBoundingQuery=SELECT MIN(id),MAX(id) FROM table
+invalidTable=table
+
+#POSTGRESQL Valid Properties
+connectionArgumentsList=[{"key":"queryTimeout","value":"-1"}]
+connectionTimeout=150
+numberOfSplits=2
+zeroValue=0
+splitByColumn=ID
+postgreImportQuery=select * from public.SOURCETABLE_dnbbbcpszj where $CONDITIONS
```

Comment:
Why are you hardcoding a table name here in the query. Try using PluginPropertyUtils.addPluginProp() in TestSetup Hooks class and create this query there with random table name each time.

Reply:
Removed.